### PR TITLE
Set background color white for smoke plots

### PR DESCRIPTION
**Pull request by kayeekayee**
Changed the lowest value colors be actual white instead of pale blue for both integrated smoke and near surface smoke.

The following plots show the changes of the actual white (left) and original pale blue color (right):

smoke_int_before_after

smoke_sfc_before_after

Please review.

### CODE REVIEW

**Review by Brian-Jamison — APPROVED** [review]
Looks good!

File: adb_graphics/specs.py [hunk 1]
```diff
@@ -320,10 +320,11 @@ def smoke_colors(self) -> np.ndarray:
 
         ''' Default color map for smoke plots. '''
 
-        blues = cm.get_cmap('Blues', 6)(range(5))
+        white = cm.get_cmap('Greys', 2)([0])
+        blues = cm.get_cmap('Blues', 15)([2,5,8,11])
```
Comment (christinaholtNOAA) [inline adb_graphics/specs.py:324]: To leave the blues just as they were before as Ravan requested, you should be able to leave it with the more straightforward:

```
blues = cm.get_cmap('Blues', 6)(range(1, 5))
```

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]
@kayeekayee Thanks for this contribution. I know Ravan will be happy to see the change finally made. 

Please make sure the linter is happy with a 10/10 score. If you need help with that, let us know. :)

File: adb_graphics/specs.py [hunk 2]
```diff
@@ -320,10 +320,11 @@ def smoke_colors(self) -> np.ndarray:
 
         ''' Default color map for smoke plots. '''
 
-        blues = cm.get_cmap('Blues', 6)(range(5))
+        white = cm.get_cmap('Greys', 2)([0])
+        blues = cm.get_cmap('Blues', 6)(range(1, 5))
         green_yellow_red = cm.get_cmap('RdYlGn_r', 18)([1, 3, 5, 9, 12, 13, 14, 16, 18])
         purple = np.array([mpcolors.to_rgba('xkcd:vivid purple')])
-        return np.concatenate((blues, green_yellow_red, purple))
+        return np.concatenate((white,blues, green_yellow_red, purple))
```
Comment (christinaholtNOAA) [inline adb_graphics/specs.py:327]: I think the linter is grumbling wanting a space between `white,` and `blues`.

**Review by christinaholtNOAA — APPROVED** [review]
Thanks for those changes, @kayeekayee! Looks like it's just down to one tiny change to pass the linter, but I'm good with merging it after that.

**Review by cshartsough — APPROVED** [review]
agree, it's nice work.  dang that linter, always keeping us honest.  :-)  looks good.

**Comment by christinaholtNOAA** [comment]
@kayeekayee Do you have the necessary permissions to merge the PR? If so, go for it! :)

If not, let me know and I will fix that.

**Pull request merged**